code/game/dna/mutations/_mutations.dm

```diff
@@ -26,8 +24,6 @@
 /datum/mutation/proc/can_activate(mob/M, flags)
 	if(flags & MUTCHK_FORCED)
 		return TRUE
-	// Probability check
-	return prob(activation_prob)
```

Comment:
Since there is no more RNG involved you want to make this proc always return TRUE at the end?

Reply:
If we are returning TRUE always, is there even a need for the MUTCH_FORCED check?
cant it be entirely
```
/datum/mutation/proc/can_activate(mob/M, flags)
     return TRUE
```
?

Reply:
I'll try this out later tonight and see if it causes any issues.

Reply:
Tested out Oyu's suggested change, everything seems to work as intended.